packages/node-sqlite3/README.md

```diff
@@ -0,0 +1,8 @@
+# database example
```

Comment:
`Node Sqlite 3 example`.

Could you also add a link for it on the main [readme](https://github.com/mochajs/mocha-examples/blob/master/README.md#examples) please 😄 ? I think its easier to see here.